Advanced/Sync Addressables/Assets/SyncAddressables/SyncBundleProvider.cs

```diff
@@ -23,10 +25,37 @@ internal void Start(ProvideHandle provideHandle)
                 Debug.LogError("the bundle failed " + provideHandle.Location.InternalId);
             provideHandle.Complete(this, m_AssetBundle != null, null);
         }
+
+        internal void Unload()
+        {
+            if (m_AssetBundle != null)
+            {
+                m_AssetBundle.Unload(true);
+                m_AssetBundle = null;
+            }
+        }
     }
     
     public override void Provide(ProvideHandle providerInterface)
     {
         new SyncAssetBundleResource().Start(providerInterface);
     }
+
+    public override void Release(IResourceLocation location, object asset)
+    {
+        if (location == null)
+            throw new ArgumentNullException("location");
+        if (asset == null)
+        {
+            Debug.LogWarningFormat("Releasing null asset bundle from location {0}.  This is an indication that the bundle failed to load.", location);
```

Comment:
This warning is redundant. Condition 'asset == null' should be silently suppressed. Error logging during bundle load should be enough for package user to track the problem.